sphinx-docs/reference.rst

```diff
@@ -0,0 +1,53 @@
+============================
+Reference and Contributors
+============================
+
+
+Citation
+========
+
+If you are using the package please cite:
+
+
+* 
+  .. image:: https://zenodo.org/badge/108329371.svg
+     :target: https://zenodo.org/badge/latestdoi/108329371
+     :alt: DOI
+
+* J. Duarte *et al.*\ , "Fast inference of deep neural networks in FPGAs for particle physics", `JINST 13 P07027 (2018) <https://dx.doi.org/10.1088/1748-0221/13/07/P07027>`_\ , `arXiv:1804.06913 <https://arxiv.org/abs/1804.06913>`_.
+
+If you are using the boosted decision tree implementation, please cite also:   
+
+
+* S. Summers *et al.*\ , "Fast inference of boosted decision trees in FPGAs for particle physics", `arXiv:2002.02534 <https://arxiv.org/abs/2002.02534>`_.
+  If you are using the binary or ternary neural network implementation, please also cite:
+* G. Di Guglielmo *et al.*\ , "Compressing deep neural networks on FPGAs to binary and ternary precision with hls4ml", `arXiv:2003.06308 <https://arxiv.org/abs/2003.06308>`_
+
+Additional Talks and Presentations
+==================================
+
+
+* eScience 2019: `talk <https://escience2019.sched.com/event/Uuiy/machine-learning-on-fpgas-for-low-latency-and-high-throughput-inference?iframe=yes&w=100%&sidebar=yes&bg=no#>`_
+* ACAT 2019: `talk <https://indico.cern.ch/event/708041/contributions/3269690/>`_
+* Zurich Hands-on Course: `course <https://indico.cern.ch/event/769727/>`_
+* TWEPP 2018: `talk <https://indico.cern.ch/event/697988/contributions/3055990/>`_
+* CHEP 2018: `talk <https://indico.cern.ch/event/587955/contributions/2937529/>`_
+* Connecting the Dots 2018: `talk <https://indico.cern.ch/event/658267/contributions/2813688/>`_
+* Fermilab Research Techniques Seminar: `talk <https://indico.fnal.gov/event/16908/>`_
+* CERN EP/IT Data Science Seminar: `talk <https://indico.cern.ch/event/721567/>`_ 
+
+Contributors
+============
+
+
+* Vladimir Loncar, Jennifer Ngadiuba, Maurizio Pierini, Sioni Summers [CERN]
+* Javier Duarte [University of California San Diego]
+* Sergo Jindariani, Benjamin Kreis, Ryan Rivera, Nhan Tran [Fermilab]
+* Edward Kreinar [Hawkeye360]
+* Song Han, Philip Harris, Dylan Rankin [MIT]
+* Zhenbin Wu [University of Illinois at Chicago]
+* Mark Neubauer [University of Illinois Urbana-Champaign]
+* Shih-Chieh Hsu [University of Washington]
+* Giuseppe Di Guglielmo [Columbia University]
+* Duc Hoang [Rhodes College]
+
```

Comment:
Should add yourself as one of the contributors.

Reply:
Ok, will do.